tools/ratbagctl.in.in

```diff
@@ -42,6 +42,14 @@ button_specials_strmap = {
 }
 
 
+def convert_str_to_evcode(s: str) -> int:
+    if not s.startswith("KEY_") and not s.startswith("BTN_"):
+        msg = "Don't know how to convert {}".format(s)
+        raise argparse.ArgumentTypeError(msg)
+
+    return evdev.ecodes.ecodes[s]
```

Comment:
Not a problem with this PR, just something we can improve along the way:
Currently this is going to raise a `KeyError` if there is no such key as the user supplied, I think it would make sense to make a more sensible error:
```suggestion
    try:
        return evdev.ecodes.ecodes[s]
    except KeyError:
        msg = "No such button or key `{}`".format(s)
        raise argparse.ArgumentTypeError(msg)
```